src/main/resources/swagger.api/schema/resource/Work.json

```diff
@@ -11,13 +11,6 @@
     {
       "type": "object",
       "properties": {
-        "responsibiltyStatement": {
-          "type": "array",
-          "items": {
-            "type": "string"
-          },
-          "x-json-property": "http://bibfra.me/vocab/marc/responsibilityStatement"
```

Comment:
Shouldn't we place it to the Instance.json under new dictionary value?

Reply:
@PBobylev yes, we should but in a separate ticket (https://folio-org.atlassian.net/browse/MODLD-364)

Reply:
I see, ok then, thx